src/options/modules/CustomOptionTriggers.js

```diff
@@ -91,13 +91,24 @@ function applyDebugMode(optionValue) {
  *
  * @function
  * @private
- * @param {string} optionValue
+ * @param {number} optionValue
  * @returns {void}
  */
 function updateQrQuietZoneStatus(optionValue) {
     const elQrQuietZoneStatus = document.getElementById("qrQuietZoneStatus");
+    let messageName = "optionQrQuietZoneStatusPlural";
 
-    elQrQuietZoneStatus.textContent = optionValue;
+    if (optionValue === 1) {
```

Comment:
Now we get to the difficult part… :wink: 

But I think currently it should be easy, but this simple if construct may be a little to easy if further translations are added, because some languages do use the singular form for other numbers, too…

See https://localization-guide.readthedocs.io/en/latest/l10n/pluralforms.html and https://github.com/PrivateBin/PrivateBin/blob/master/js/privatebin.js#L439-L465 for an example implementation.

However, for now, you obviously do not need to implement support for all languages. It is enough, if you just add a function `getPluralForm` or similarly named that is used here and may be extended later for some languages.

Note I am also no expert here, but this is a problem I've just remembered. :wink:

Reply:
Combined with the "maybe use a simple translation for the singular", I guess this gets hard. So, IMHO, you can skip the part with the "use an extra hardtranslated translation for one item"; or you just add another translation for "one, and really one".

Reply:
I added the `getPluralForm` function, which returns the message translation names. In the future, a string of all possible plural forms could be added to the lang files, separated by `;`. Then the function would just return an integer, indicating which form in the split array the word should take.

But for now, hopefully this should be fine with the languages currently translated.

Reply:
Yeah, I'd say that's okay for now.